Signal/src/contact/OWSContactsManager.m

```diff
@@ -107,9 +109,7 @@ - (void)doAfterEnvironmentInitSetup {
 }
 
 - (void)verifyABPermission {
-    if (!self.addressBookReference) {
```

Comment:
We can't debounce here due to concurrency. We need to do it inside `setupAddressBookIfNecessary`, using the `ADDRESSBOOK_QUEUE` to avoid race conditions.